packages/core-components/src/components/DependencyGraph/DependencyGraph.tsx

```diff
@@ -162,6 +163,14 @@ export interface DependencyGraphProps<NodeData, EdgeData>
    * Default: `enabled`
    */
   zoom?: 'enabled' | 'disabled' | 'enable-on-click';
+  /**
+   * A factory for curve generators addressing both lines and areas.
+   *
+   * @remarks
+   *
+   * Default: 'curveMonotoneX'
+   */
+  curve?: Curve;
```

Comment:
Think that we can just define the type inline here rather than exporting a new one for it. Something like `curve?: 'curveStepBefore' | 'curveMonotoneX'` should be fine! :pray:

Reply:
@benjdlambert Of course! I have just changed it.